lti_consumer/settings.py

```diff
@@ -0,0 +1,81 @@
+"""
+Django settings for xblock-lti-consumer project.
+
+These settings are used when compiling translations.
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+from __future__ import absolute_import
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
```

Comment:
Are you planning to "productionize" these settings?

Reply:
these are solely used to run translation compilation. I can make that a bit clearer in the docstring

Reply:
I moved this entire file under the locale folder so you don't even see it unless you're messing with translations.